tests/DependencyEmitter/Fixtures/FQDNNamespacePrefix/Uses/Foo.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace FQDNNamespacePrefix\Uses;
+
+use FQDNNamespacePrefix\FQDN;
```

Comment:
Is it intentional, that this does not match with the file path?

```suggestion
use FQDNNamespacePrefix\Uses;
```

Reply:
I don't understand, it matches: 
* `tests/DependencyEmitter/Fixtures/FQDNNamespacePrefix/Uses/Foo.php` => `namespace FQDNNamespacePrefix\Uses;`
* `tests/DependencyEmitter/Fixtures/FQDNNamespacePrefix/FQDN.php` => `namespace FQDNNamespacePrefix;`

The use line 5 imports `tests/DependencyEmitter/Fixtures/FQDNNamespacePrefix/FQDN.php`.

This test validates that the class `FQDNNamespacePrefix\FQDN` is correctly counted as a dependency even though it is also a namespace prefix for `FQDNNamespacePrefix\FQDN\SomeClass`.

Reply:
Sorry, it was too early. I was looking at the use not the namespace. Nevermind